src/dataviews/histogram-dataview-model.js

```diff
@@ -401,7 +413,10 @@ module.exports = DataviewModelBase.extend({
 
   _onTotalsError: function (model, error) {
     var parsedError = error && this._parseError(error);
-    this._triggerStatusError(parsedError);
+
+    if (parsedError && parsedError.message !== 'abort') {
```

Comment:
Why do we check if error is distinct from `abort`?

Reply:
Because when you send the same request twice one of them gets aborted and it was showing an alert here: https://github.com/CartoDB/carto.js/blob/6c0c79f0987fd770bccf785380a667600ddbdde0/examples/public/dataviews/histogram.html#L76

We're doing that check in more places in the code, although with the debounce we merged I'm not sure if this will keep happening.

Reply:
Can we check that it is not happening anymore to remove that if? 🙏

Reply:
Still happening